src/lib/viewers/controls/icons/IconVolumeLow24.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+
+function IconVolumeLow24(props: React.SVGProps<SVGSVGElement>): JSX.Element {
+    return (
+        <svg focusable={false} height={24} viewBox="-3 5 24 24" width={24} {...props}>
+            <path d="M0 14v6h4l5 5V9l-5 5H0z" fill="#FFF" />
```

Comment:
Should these `#fff` and `#FFF` be consistent?

Reply:
The other icons are already inconsistent. I can do this separately if it's important?

Reply:
Should we start using the buie `$white` color?